internal/config/profile.go

```diff
@@ -573,7 +569,31 @@ func (p *Profile) Save() error {
 	return viper.WriteConfigAs(p.Filename())
 }
 
-func configHome() (string, error) {
+// ReadMongoCLIConfig retrieves mongoCLI configuration.
+func ReadMongoCLIConfig() string { return p.readMongoCLIConfig() }
+func (p *Profile) readMongoCLIConfig() string {
```

Comment:
`ReadMongoCLIConfig` names hint at loading the file into memory, which this method doesn't seem to be doing. Your only use of `p` is to get the file system wrapper, `ReadMongoCLIConfig` and `readMongoCLIConfig` are not used is this package, does this need to be in this package?

Reply:
updated